psalm.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<psalm
+        errorLevel="2"
+        phpVersion="8.2"
```

Comment:
What's the rationale for choosing 8.2 here, and 8.3 for PHPStan?

Reply:
To be honest, I have little experience of which version to run these tests on. I see on the ORM project that it is still on 8.2, @derrabus here suggested 8.3 for the stan, but I was careless with the psalm config. For doctrine1, for example, the lowest supported version is used (7.4). In my own projects, I try to choose the highest version. I'll upgrade to 8.3.